addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -67,7 +69,11 @@ def _compute_name(self):
         self.filtered(
             lambda x: x.journal_id.l10n_latam_use_documents and x.l10n_latam_document_type_id
             and not x.l10n_latam_manual_document_number and x.state == 'draft' and not x.posted_before).name = '/'
-        super(AccountMove, self - without_doc_type - manual_documents)._compute_name()
+        group_by_document = defaultdict(list)
```

Comment:
While this looks elegant, you shouldn't forget that `records + records` is done by converting the internal list of ids.
https://github.com/odoo/odoo/blob/d2cb61dd8487467c38ba46d083b25d815155349a/odoo/models.py#L5963-L5979

This makes the `+=` operator `O(n+m)` with `n` and `m` the size of the left and right operands.
And it finally means `O(n^2)` in the loop to build the dict values (since `m` is always `1` in that case).
Small operation, but worth keeping in mind as it might have an effect for big recordsets :wink: